src/components/Checkout/BasketItem.vue

```diff
@@ -191,6 +191,11 @@ export default {
 			return this.loan.team ? this.loan.team.id : 0;
 		}
 	},
+	watch: {
+		teams() {
```

Comment:
Looking at `ThanksPage`, it looks like teams should be prefetched, so I wouldn't think a `watch` was needed. Is that not the case?

Reply:
Okay makes sense. Using the `watch` isn't a bad thing...